packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/CombinationGraph/Regions/Simple.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/lib/components/Line/LineGraph.module.scss';
+import '@cerner/terra-graphs/lib/components/PairedResult/PairedResultGraph.module.scss';
+import lineAndPairedResultConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/lineAndPairedResultConfig';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineAndPairedResultData';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const graphConfig = utils.deepClone(lineAndPairedResultConfig('#lineWithPairedResultRegion'));
+const dataset = utils.deepClone(data);
+
+dataset[0].regions = [
+  {
+    axis: 'y',
+    start: 6,
+    end: 18,
+  },
+];
+dataset[1].regions = {
+  high: [
+    {
+      axis: 'y2',
+      start: 120,
+      end: 180,
+      color: '#c8cacb',
+    },
+  ],
+  low: [
+    {
+      axis: 'y2',
+      start: 20,
+      end: 80,
```

Comment:
Does low also need a color like high?

Reply:
It is optional ben if the consumer does not provide any color then our default color(black) will come into the picture.